src/components/implementation/tests/unit_cxx/Makefile

```diff
@@ -0,0 +1,18 @@
+C_OBJS=cxx.o
+COMPONENT=cxx_test.o
+INTERFACES=
+DEPENDENCIES=cbuf_mgr printc sched valloc mem_mgr_large torrent evt timed_blk lock
```

Comment:
These are a relic of the previous implementation.  We likely don't want them anymore.